### PR TITLE
docs(connect-explorer): connect-deeplink / mobile

### LINKED ISSUE #14533: Create @trezor/connect-deeplink section in Explorer
https://www.figma.com/design/JqchtWljVt5DY3ici2DWdC/Connect-Docs?node-id=781-19&t=XkL2Oxl8aOMjYtA7-4

### PR DESCRIPTION
**Pull request by martykan**
## Description

Add a section about mobile to the index page, including quickstart guide.
Add a page documenting the deeplink URL spec.

Add the option to open and receive deeplinks from Connect Explorer.

Link: https://dev.suite.sldev.cz/connect/doc/native-connect-popup/

## Related Issue

Resolve #14533

### CODE REVIEW

File: packages/connect-explorer/src/actions/trezorConnectActions.ts [hunk 1]
```diff
@@ -160,20 +163,39 @@ export const init =
...
+                await TrezorConnectDeeplink.init({
+                    ...connectOptions,
+                    deeplinkOpen(url) {
+                        window.open(url, '_blank');
+                    },
+                    deeplinkCallbackUrl:
+                        (process.env.CONNECT_EXPLORER_FULL_URL || window.location.origin) +
+                        '/callback',
+                });
+                const bc = new BroadcastChannel('trezor_connect_callback');
```
Comment (Nodonisko) [inline packages/connect-explorer/src/actions/trezorConnectActions.ts:176]: what is this good for? you are not passing `bc` anywhere so I guess no one is subscribed to that messages anyway

Comment (martykan) [reply packages/connect-explorer/src/actions/trezorConnectActions.ts:176]: The callback page opens a BroadcastChannel with the same ID and through this it's able to send the response back to the main tab.

Comment (Nodonisko) [reply packages/connect-explorer/src/actions/trezorConnectActions.ts:176]: Thanks for explanation, I was not familiar with that API

File: packages/connect-mobile/src/index.ts [hunk 2]
```diff
@@ -123,6 +123,11 @@ export class TrezorConnectDeeplink implements ConnectFactoryDependencies {
             return;
         }
 
+        if (!this.messagePromises[id]) {
+            // Most likely old ID, ignore
+            return;
+        }
+
```
Comment (mroz22) [inline packages/connect-mobile/src/index.ts:130]: feels like it would deserve a standalone commit maybe. nitpick.

**Review by mroz22 — COMMENTED** [review]
image
 
on sldev

**Comment by mroz22** [comment]
https://github.com/trezor/trezor-suite/tree/develop/packages/connect-examples/deeplink-expo not found

File: packages/connect-explorer/src/actions/trezorConnectActions.ts [hunk 3]
```diff
@@ -143,6 +145,7 @@ export const init =
         // Get default coreMode from URL params (?core-mode=auto)
         const urlParams = new URLSearchParams(window.location.search);
         const coreMode = (urlParams.get('core-mode') as ConnectOptions['coreMode']) || 'auto';
+        const deeplink = urlParams.get('deeplink') === 'true'; // TODO
```
Comment (mroz22) [inline packages/connect-explorer/src/actions/trezorConnectActions.ts]: I imagine, that in the end, deeplink will be only one of the core modes maybe (only available if some conditions are met, such as registering callbacks etc).

**Review by mroz22 — APPROVED** [review]
just some nitpicks. I think it makes sense to have this already in develop.

**Review by mroz22 — APPROVED** [review]
